ch_pipeline/processing/daily.py

```diff
@@ -134,10 +141,25 @@
       params:
         exclude_intracyl: true
 
+    # Identify individual baselines with much lower weights than expected and
+    # mask out the entire time-freq sample
+    - type: draco.analysis.flagging.ThresholdVisWeightTime
+      in: tstream
+      out: bad_baseline_mask
+
+    # Identify decorrelated cylinders
+    - type: ch_pipeline.analysis.flagging.MaskDecorrelatedCylinder
+      requires: freqmap
+      in: [tstream, inputmap]
+      out: decorr_cyl_mask
+      params:
+        threshold: 5.0
+        max_frac_freq: 0.1
+
     # Average over redundant baselines across all cylinder pairs
     - type: draco.analysis.transform.CollateProducts
       requires: manager
-      in: tstream
+      in: tstream_bbm
```

Comment:
Since the bad-baseline masking was moved down in the pipeline, this should be changed back:
```suggestion
      in: tstream
```

Reply:
Thanks. Actually I've fixed that one locally as I wasn't getting any output!